app/controllers/sponsor/contributions_controller.rb

```diff
@@ -8,41 +8,25 @@ def index
   end
 
   def show
-    # @contribution made available with private method
   end
 
-  # As of now sponsor should not be able to create contributions
-  # def new
-  #   @contribution = Contribution.new
-  #   # authorize @contribution
-  # end
-
-  # def create
-  #   @contribution = Contribution.new(contribution_params)
-  #   @contribution.user = contribution_user
-  #   # authorize @contribution
-  #   if @contribution.save
-  #     redirect_to contribution_path(@contribution)
-  #   else
-  #     render :new
-  #   end
-  # end
-
   def edit
```

Comment:
You can just remove this edit method as a whole

Reply:
ok deleted